shared/js/ui/models/top-blocked.es6.js

```diff
@@ -20,17 +20,17 @@ TopBlocked.prototype = window.$.extend({},
       return new Promise((resolve, reject) => {
         this.fetch({getTopBlockedByPages: this.numCompanies})
           .then((data) => {
+            // TODO remove this before merging - added for testing
+            this.addTestData()
+            return resolve()
```

Comment:
We may want to just comment this out. Developing the top trackers views is a headache if you want to check the views with and without trackers, so I can see this being useful in the future.